src/ralph/models/xapi/fields/object.py

```diff
@@ -0,0 +1,60 @@
+"""Common xAPI object field definitions"""
+
+from typing import Optional
+
+from pydantic import AnyUrl, Field
+
+from ..config import BaseModelWithConfig
+from ..constants import (
+    EN,
+    PAGE,
+    XAPI_ACTIVITY_PAGE,
+    XAPI_EXTENSION_COURSE,
+    XAPI_EXTENSION_MODULE,
+    XAPI_EXTENSION_SCHOOL,
+)
+
+
+class PageObjectDefinitionExtensionsField(BaseModelWithConfig):
+    """Represents the `object.definition.extensions` xAPI field for page viewed statement.
+
+    Attributes:
+        school (str): Consists of the name of the school.
+        course (str): Consists of the name of the course.
+        module (str): Consists of the name of the module.
+    """
+
+    school: Optional[str] = Field(alias=XAPI_EXTENSION_SCHOOL)
+    course: Optional[str] = Field(alias=XAPI_EXTENSION_COURSE)
+    module: Optional[str] = Field(alias=XAPI_EXTENSION_MODULE)
+
+
+class PageObjectDefinitionField(BaseModelWithConfig):
+    """Represents the `object.definition` xAPI field for page viewed xAPI statement.
+
+    WARNING: It doesn't include the recommended `description` field nor the optional `moreInfo`,
+    `Interaction properties` and `extensions` fields.
+
+    Attributes:
+       type (str): Consists of the value `http://activitystrea.ms/schema/1.0/page`.
+       name (dict): Consists of the dictionary `{"en": "page"}`.
+       extensions (dict): See PageObjectDefinitionExtensionsField.
+    """
+
+    name: dict[EN, PAGE] = {EN.__args__[0]: PAGE.__args__[0]}
```

Comment:
This would be great) Unfortunately it doesn't make the same result :disappointed: 
```
>>> from typing import Literal
>>> test = Literal["Test"]
>>> str(test)
"typing.Literal['Test']"
>>> test.__args__[0]
'Test'
```

Reply:
That's unfortunate.